tests/robot-tests/tests/libs/common.robot

```diff
@@ -23,7 +23,20 @@ do this on failure
     capture large screenshot and html
     set selenium timeout  3
 
+custom testid locator strategy
+  [Arguments]  ${browser}  ${test_id}   ${tag}  ${constraints}
+  Log to console  ${tag}
+  Log to console  ${constraints}
+  Log to console  ${test_id}
```

Comment:
These log statements can be removed

Reply:
Cool, I left that in just in case this selector became flaky